src/units/abilities.cpp

```diff
@@ -1963,6 +1965,15 @@ effect::effect(const unit_ability_list& list, int def, const_attack_ptr att, boo
 			}
 		}
 
+		if((type == "resistance")){
+			if(cfg.has_attribute("max_value")){
+				max_value = max_value ? std::max(*max_value, cfg["max_value"].to_int()) : cfg["max_value"].to_int();
+			}
+			if(cfg.has_attribute("min_value")){
+				min_value = min_value ? std::min(*min_value, cfg["min_value"].to_int()) : cfg["min_value"].to_int();
+			}
+		}
```

Comment:
This block of code is a replacement for the code removed from `unit.cpp`, correct?

Reply:
Yes with addition of min_value